content/en/community/hackathons/sessions/baby-steps/work/02-rot13/launch.sh

```diff
@@ -1,24 +1,24 @@
 #!/bin/bash
```

Comment:
I suggest we have only one `launch.sh` file and make a symlink to it for the rest of the exercises?

Reply:
I think we should keep separate files for this. Students might alter the directory hierarchy when copying the materials.